### DISCUSSION TITLE
After a certain version update, When i press `/` to search for content and use the delete key to delete to the last character, the shadow of the character remains.

### DISCUSSION THREAD

**Discussion by gengdz** [LazyVim/LazyVim#4400] [Q&A]
1. Press `/` to search for `abc`.
      ![image](https://github.com/user-attachments/assets/e2ab9fdb-2695-41ab-8aa8-474e7bf09bd8)
      
2. Use the delete key 3 times to delete `abc`.
    delete `bc`
![image](https://github.com/user-attachments/assets/b41112c2-4c6b-42fc-82fb-281b12a884fb)

3. Notice that the letter `a` below was not deleted.
 delete `a`
    ![image](https://github.com/user-attachments/assets/89799e72-4c6f-46c8-988a-acb136eda1ef)

If I enter other text at this point, for example `name`, it will actually take effect and not become `aname`.

If I directly use `c-w` to delete the entire `abc`, the situation will change to the following (the text content is still there, but it appears similar to a placeholder)

![image](https://github.com/user-attachments/assets/14f78c89-1bf5-49b6-966d-8e42efe4fb09)

my config: https://github.com/gengdz/dotfiles/blob/main/nvim/lua/config/autocmds.lua

**Comment by dpetka2001** [comment]
https://github.com/folke/noice.nvim/issues/892
